indico/modules/events/surveys/schemas.py

```diff
@@ -0,0 +1,30 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2023 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from marshmallow.fields import Function, List, Nested
```

Comment:
I prefer `from marshmallow import fields` and using `fields.Whatever` in the schemas (it's mainly pretty old code that uses tons of individual type imports ;0)